### DISCUSSION TITLE
sending metric with json file

### DISCUSSION THREAD

**Discussion by therus000** [prometheus/pushgateway#434] [Q&A]
good day 
what the way to send the metric with JSON file
what JSON file should formated?
is there and example?

**Comment by beorn7** [comment]
Pushgateway only supports delimited binary protocol buffers or the simple flat text format (the latter being used in the examples in the README.md).

You could use the Pushgateway support in many of the [instrumentation libraries](https://prometheus.io/docs/instrumenting/clientlibs/) to write a program receiving JSON, converting it to Prometheus format, and pushing it to the Pushgateway.
